test/main.sh

```diff
@@ -55,11 +55,21 @@ if [ "${PWD}" != "$(dirname "${0}")" ]; then
 fi
 import_subdir_files includes
 
+# is_backend_available checks if a given backend is available by matching it against the list of available storage backends.
+# Surrounding spaces in the pattern (" $(available_storage_backends) ") are used to ensure exact matches,
+# avoiding partial matches (e.g., "dir" matching "directory").
+is_backend_available() {
+    case " $(available_storage_backends) " in
+        *" $1 "*) return 0;;
```

Comment:
That's a terrible hack to emulate `grep -w` ... but I like it nevertheless ;)
Thanks!